chaosaws/ecs/probes.py

```diff
@@ -1,2 +1,25 @@
 # -*- coding: utf-8 -*-
-__all__ = []
+from chaoslib.exceptions import FailedActivity
+from chaoslib.types import Configuration, Secrets
+
+from chaosaws import aws_client
+
+__all__ = ["service_is_not_deploying"]
+
+
+def service_is_not_deploying(cluster: str,
```

Comment:
I'm wondering if this method name should be reversed. I have use cases where I want the service deploying, and I have them where I don't.